rfcs/onboard_infrastructure_providers.md

```diff
@@ -0,0 +1,48 @@
+### Introduction
+
+It seems reasonable that being Paseo a community testnet not a single party has to make all the efforts of maintaining this platform. At the same time, moving away from a single operator should not compromise the availablity of the network.
```

Comment:
A Paseo is a decentralised community testnet, we want to open the opportunity for validators in the ecosystem to contribute to the success and scaling of the network.